ArduCopter/mode_plancktracking.cpp

```diff
@@ -67,33 +69,46 @@ void ModePlanckTracking::run() {
                   direction,
                   relative_angle);
           }
-          paylod_yaw_rate = false;
+          is_paylod_yaw_rate = false;
         }
         else if(!mount->has_pan_control() || mount->mount_yaw_follow_mode == AP_Mount::gimbal_yaw_follows_vehicle)
         {
           //in this case, APM is getting yaw rate commands from the tablet via mavlink,
           //so we set the set the fixed yaw command to this rate, and tell the
           //angle controller in mode guided to interpret as a rate. Note that the
           //auto yaw will be set to mode fixed, though it is actually controlling rate.
-          paylod_yaw_rate = true;
+          is_paylod_yaw_rate = true;
 
-          //If we've stopped getting the mavlink yaw rate commands, use 0 yaw rate command
-          if(AP_HAL::micros64() - mount->get_last_mount_control_time_us() > 500000)
+          if(copter.mode_guided.mode()==Guided_Angle)
           {
-            copter.flightmode->auto_yaw.set_fixed_yaw(
-                  0.0f,
-                  0.0f,
-                  0,
-                  0);
+            //If we've stopped getting the mavlink yaw rate commands, use 0 yaw rate command
+            if(AP_HAL::micros64() - (mount->get_last_mount_control_time_us() > 500000))
```

Comment:
Looks like parenthesis are weird here; you're subtracting a boolean from a 64-bit

Reply:
fixed